docs/services/vision/classification.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Classification (or 2D image classification)"
+linkTitle: "Classification"
+weight: 20
+type: "docs"
+description: "Select an algorithm that outputs a class label and confidence score associated with a 2D image."
+tags: ["vision", "computer vision", "CV", "services", "classification"]
+# SMEs: Bijan, Khari
+---
+
+_2D Image Classification_ is the process of taking a 2D image from a camera and deciding which class label, out of many, best describes the given image.
+Any camera that can return 2D images can use 2D image classification.
+
+Which class labels may be considered for classification varies and will depend on the machine learning model and how it was trained.
```

Comment:
```suggestion
The class labels used for classification vary and depend on the machine learning model and how it was trained.
```
Not sure whether this is correct, but the current sentence seems a bit awk and vague--"may be considered" seems ambiguous. Is it that some models are unable to consider some labels due to features of the class? Or is it that some models just haven't been trained on certain classes?